test/support/verifiers/ResultsPageVerifier.ts

```diff
@@ -1,23 +1,23 @@
 export default class ResultsPageVerifier {
   resultsDataCorrect(): void {
     const correctFileInfo: string[] = [
-      'Sonarqube Java Heimdall_tools Sample',
-      '1.3.0',
-      'Heimdall Tools1.3.0',
-      'null'
+      'Acme Overlay Example',
+      '4.19.2',
+      'mac_os_x19.4.0',
+      '0.605412'
     ];
     const correctProfileInfo: string[] = [
-      '7.9.1.27448',
-      'Sonarqube Java Heimdall_tools Sample',
-      'Thu,26 Sep 2019 10:56:37',
-      '6f94b370ebaa414fda5a2a80575cc772e3d0cc58ce04adc6d74a61a48080c886',
-      'SonarQube Scan of Project: heimdall'
+      '0.2.0',
+      'Acme Overlay Example',
+      '2020-07-06T17:54:48-04:00',
+      '6414f43b3c6ff3106c96b2e75067a9ed6144a728373b864f4de8b3fb3c6c528f',
+      'Meta profile for Acme Inc'
     ];
     const correctCards: string[] = [
-      'Passed: 0',
-      'Failed: 3',
-      'Not Applicable: 0',
-      'Not Reviewed: 0'
+      'Passed: 5',
+      'Failed: 60',
+      'Not Applicable: 1',
+      'Not Reviewed: 35'
```

Comment:
Could you elaborate on the reason for this change?

Reply:
The check for results data loads the first available sample, I changed the v-data-table to sort by filename automatically, so a different file is loaded during tests